BankSDK/GiniBankSDK/Sources/GiniBankSDK/Core/Skonto/DocumentPages/DocumentPagesViewModel.swift

```diff
@@ -18,7 +18,7 @@ struct CornerBoundingBoxes {
     var bottomRight: ExtractionBoundingBox
 }
 
-final class DocumentPagesViewModel {
+final class DocumentPagesViewModel: DocumentPagesViewModelProtocol {
```

Comment:
[suggestion] I suggestion to rename this view model since is for Skonto like we did for TD: SkontoDocumentPagesViewModel